addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-ascanconfig.html

```diff
@@ -44,6 +44,7 @@ <H2>YAML</H2>
         enabled:                           # Bool: If enabled. Default: false
         encodeCookieValues:                # Bool: If cookie values should be encoded. Default: false
       scripts:                             # Bool: If Input Vector scripts should be used. Default: true
+    excludePaths:                          # An optional list of regexes to exclude
```

Comment:
Worth documenting somewhere that existing ones are lost, or use `ExtensionActiveScan.setExcludeList` (though same applies for this one it might be less surprising since this is used just programmatically).